cpu/stm32l4/include/cpu_conf.h

```diff
@@ -36,7 +36,7 @@ extern "C" {
  * @{
  */
 #define CPU_DEFAULT_IRQ_PRIO            (1U)
-#define CPU_IRQ_NUMOF                   (82U)
+#define CPU_IRQ_NUMOF                   (81U)
```

Comment:
Not sure that it makes a big difference on a CPU with 1MB of flash but I accept the argument. I'll update this with preprocessor macros in #6838.